lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -21,10 +21,21 @@ class BaseI18nTestCase(TestCase):
     """
     Base utilities for i18n test classes to derive from
     """
+    def __init__(self, *args, **kwargs):
```

Comment:
Typically, we don't create `__init__` methods for Test classes.  Instead, we use the `setUp` and `setUpClass` methods for initialization code.